<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat</class>
 <widget class="QWidget" name="chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 245, 245);</string>
  </property>
  <widget class="QListWidget" name="friendsListWidget">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>0</y>
     <width>312</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 234, 232);
border-color: rgb(231, 230, 229);</string>
   </property>
  </widget>
  <widget class="QListWidget" name="toolListWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>72</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(42, 42, 42);</string>
   </property>
  </widget>
  <widget class="QWidget" name="topWidget" native="true">
   <property name="geometry">
    <rect>
     <x>382</x>
     <y>0</y>
     <width>694</width>
     <height>73</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 245);
border-color: rgb(245, 245, 245);</string>
   </property>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>642</x>
      <y>0</y>
      <width>53</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="maxSizeButton">
    <property name="geometry">
     <rect>
      <x>591</x>
      <y>0</y>
      <width>53</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>□</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minSizeButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>53</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>─</string>
    </property>
   </widget>
   <widget class="QPushButton" name="takingToolButton">
    <property name="geometry">
     <rect>
      <x>642</x>
      <y>30</y>
      <width>53</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>•••</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="bottomWidget" native="true">
   <property name="geometry">
    <rect>
     <x>382</x>
     <y>619</y>
     <width>694</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(245, 245, 245);</string>
   </property>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>52</y>
      <width>661</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;方正姚体&quot;;
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'方正姚体'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>142</y>
      <width>84</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Send(S)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendFileButton">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="emojiButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="phoneButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="addfriendButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(89, 89, 89);
font: 25 17pt &quot;等线 Light&quot;;
</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tolkingButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>33</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="friendButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>33</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="collectionButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>33</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="settingButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>759</y>
     <width>33</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>382</x>
     <y>72</y>
     <width>694</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchLineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>235</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="userPhotolabel">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>10</y>
     <width>40</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
